lib/ecs_deploy/configuration.rb

```diff
@@ -8,14 +8,16 @@ class Configuration
       :deploy_wait_timeout,
       :ecs_service_role,
       :ecs_wait_until_services_stable_max_attempts,
-      :ecs_wait_until_services_stable_delay
+      :ecs_wait_until_services_stable_delay,
+      :ecs_client_params
 
     def initialize
       @log_level = :info
       @deploy_wait_timeout = 300
       # The following values are the default values of Aws::ECS::Waiters::ServicesStable
       @ecs_wait_until_services_stable_max_attempts = 40
       @ecs_wait_until_services_stable_delay = 15
+      @ecs_client_params = { retry_mode: "standard", max_attempts: 10 }
```

Comment:
[Q] Why do you change the default retry mode?

Reply:
I intent that ecs_deploy retry as default for Repro's use case. But ecs_deploy has parameter to override initialize of ECS::Client, so I realize it's enough use default retry_mode.